rest/route/host_create.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+var createdOrCreatingHostStatuses = append(evergreen.IsRunningOrWillRunStatuses, evergreen.HostUninitialized)
```

Comment:
how do you feel about this name haha

Reply:
The var name seems fine to me!

Also, the slice append has to be written like this:
```Go
createdOrCreatingHostStatuses = append([]string{evergreen.HostUninitialized}, evergreen.IsRunningOrWillRunStatuses...)
```
If you append to the global var `IsRunningOrWillRunStatuses` slice, the [Go language spec](https://go.dev/ref/spec#Appending_and_copying_slices) says `append` will attempt to mutate the original slice to append the item, unless that slice doesn't have sufficient capacity (in which case it's forced to make a copy and append to that). This is a common Go gotcha. An easy workaround is to append to a newly-allocated slice like above.

[This blog post](https://go.dev/blog/slices-intro) goes into more detail on slices and common pitfalls.